lib/package.rb

```diff
@@ -12,7 +12,7 @@ class Package
   boolean_property :arch_flags_override, :conflicts_ok, :git_clone_deep, :git_fetchtags, :gnome, :is_fake, :is_musl, :is_static,
                    :no_compile_needed, :no_compress, :no_env_options, :no_fhs, :no_git_submodules,
                    :no_links, :no_lto, :no_patchelf, :no_shrink, :no_source_build, :no_strip, :no_zstd, :patchelf,
-                   :print_source_bashrc, :run_tests
+                   :print_source_bashrc, :run_tests, :deprecated
```

Comment:
Can we add this in alphabetical order?  The reason is to avoid potential duplicates.